core/chains/evm/logpoller/log_poller.go

```diff
@@ -944,19 +945,23 @@ func (lp *logPoller) findBlockAfterLCA(ctx context.Context, current *evmtypes.He
 	return nil, rerr
 }
 
-// pruneOldBlocks removes blocks that are > lp.ancientBlockDepth behind the latest finalized block.
```

Comment:
Guess this is form a previous PR, but we should update `lp.ancientBlockDepth` in the comment to `lp.keepFinalizedBlocksDepth`